src/pages/Account.tsx

```diff
@@ -120,159 +178,118 @@ function Account() {
+          </FlexColumn>
+          <FlexColumn $gap="0.5rem">
+            <Label $required>Affiliation</Label>
+            <Select
+              {...register('group', { required: 'Affiliation is required' })}
+              defaultValue={''}
+            >
+              <option value="" disabled>
+                Please choose an affiliation
+              </option>
+              {groups?.map((group) => (
+                <option key={group.id} value={group.id}>
+                  {group.name}
+                </option>
+              ))}
+            </Select>
+            <ErrorText>{errors.group?.message}</ErrorText>
+          </FlexColumn>
+          <Label $required>Credentials</Label>
+          {fields.map((field, i) => (
+            <FlexColumn $gap="0.5rem" key={field.id}>
+              <FlexRow $gap="0.5rem" $alignItems="center">
+                <Select
+                  {...register(`userAttributes.credentialsGroup.${i}.credential` as const, {
+                    required: 'Credential is required',
+                  })}
+                  defaultValue={''}
+                >
+                  <option value="" disabled>
+                    Please choose a credential
+                  </option>
+                  {ACADEMIC_CREDENTIALS.map((credential) => (
+                    <option key={credential} value={credential}>
+                      {credential}
                     </option>
-                    {ACADEMIC_CREDENTIALS?.map((name, idx) => (
-                      <option key={idx} value={name}>
-                        {name}
-                      </option>
-                    ))}
-                  </Select>
-                  <ErrorText>{field.state.meta.errors}</ErrorText>
-                </FlexColumn>
-              )}
-            />
-            <FlexRow $alignSelf="flex-end">
-              <Subscribe
-                selector={(state) => [state.canSubmit, state.values]}
-                children={([canSubmit, values]) => {
-                  const isEqual = JSON.stringify(values) === JSON.stringify(initialUser);
-                  return (
-                    <Button type="submit" disabled={isEqual || !canSubmit}>
-                      Submit
-                    </Button>
-                  );
-                }}
-              />
-            </FlexRow>
+                  ))}
+                </Select>
+                <Input
+                  {...register(`userAttributes.credentialsGroup.${i}.institution` as const, {
+                    required: 'Institution is required',
+                  })}
+                  placeholder="Institution"
+                />
+                <Input
+                  {...register(`userAttributes.credentialsGroup.${i}.field` as const, {
+                    required: 'Field is required',
+                  })}
+                  placeholder="Field"
+                />
+                <Button variant="text" onClick={() => remove(i)}>
```

Comment:
Is there a way to only show the remove button to the added fields? and maybe making it a trash bin icon will save us some space on mobile as it is a bit broken atm

Same with the add new credential button, I think it should only be shown after the user filled the inputs for credential, institution and field.

Reply:
We might need a FlexRow that turns into Column at 600px (our breakpoint)